### PR TITLE
Initial recipe for ocgis v1.3.0

### PR DESCRIPTION
**Pull request by bekozi**
Initial recipe for ``ocgis`` v1.3.0. I enabled osx and windows builds, and I'd like to run it through the system to check for issues before tagging the release. Once tagged, I will update the recipe before asking for a merge. :smile: I'll also remove ``krb5`` as a dependency once we get that figured out...

### CODE REVIEW

File: ocgis/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-        - libgdal <2
-        - netcdf4 <=1.1.6
+        - cf_units
+        - fiona ==1.6.3
+        - netcdf4
         - numpy
         - shapely
-        - proj.4
         - rtree
+        - krb5  # TODO (bekozi): Remove when libcom_err.so.3 issue fixed.
```
Comment (ocefpaf) [inline ocgis/meta.yaml]: I must fix that in gdal. But leave it here as a workaround until I do so.

**Comment by ocefpaf** [comment]
Thanks @bekozi! BTW we need to talk about conda-forge. I will contact you once I get back.

**Comment by bekozi** [comment]
I hope you are having a good trip! I don't know much about conda-forge, so yes, I'd like to learn more.

**Comment by bekozi** [comment]
@ocefpaf, do you have any advice for dealing with the ``GDAL_DATA`` environment variable on windows? Tests are failing because it cannot find the EPSG files: https://ci.appveyor.com/project/comtbot/conda-recipes/build/1.0.1976/job/isgfrjftbj72jp5b#L1323.

I have somewhat of a hack that works well for Linux-based systems, but it does not appear to work on windows. I noticed you have activate/deactivate triggers set up for ``gdal`` on Linux: https://github.com/ioos/conda-recipes/blob/master/gdal/posix/activate.sh#L2. Is there an equivalent for windows? Thanks!

**Comment by ocefpaf** [comment]
> Is there an equivalent for windows?

There is!  I am in the process of review out `gdal` package. (There are a few issue surfacing with naming, `libcom_err`, `libjson`, and moving to conda-forge.)

But I will get back to you soon.  Stay tuned...

**Comment by bekozi** [comment]
Okay! I'll be lurking... :smile:

**Comment by bekozi** [comment]
@ocefpaf, the checks look good! Is there any reason for me to wait before tagging? I saw you had some ``gdal`` changes incoming. I'm planning to do a squash and new pull request at the very least...

**Comment by ocefpaf** [comment]
Sorry @bekozi. I am a little bit overwhelmed this week. (Please keep pinging me until I respond!)

I am having trouble getting conda-forge's gdal build here, see #800, but nothing prevents me of merging this now. Are you done with this PR (no need to squash)?

**Comment by bekozi** [comment]
Good luck with your week! I'd like to have the recipe point to a tag. Let me do that, and I'll ping you when it's ready.

**Comment by ocefpaf** [comment]
@bekozi that AppVeyor failure is because I activated a rolling builds. (See http://help.appveyor.com/discussions/problems/3555-rolling-builds-not-very-useful.)

It is disappointing that AppVeyor cancels all builds rather than just those in the same branch :disappointed: 

I will re-start that soon, but if you are done with the changes I can just merge as-is. (We know that AppVeyor will pass based on the past builds.)

**Comment by bekozi** [comment]
I'll take your word for it. :relieved: 

Nothing changed since the last build, so yeah, feel free to merge. Thanks!

**Comment by ocefpaf** [comment]
:tada:

**Comment by bekozi** [comment]
:clap:

**Pull request merged**